### DISCUSSION TITLE
Wishing for a SIMPLE Backup Process

### DISCUSSION THREAD

**Discussion by gsownsby** [photoprism/photoprism#3100] [Q&A]
As a very new user to PhotoPrism, I have loaded about 500 of my media files (150,000+) for testing, evaluation, and play.  However, I'm a bit reluctant to load mass quantities without some sort of a reliable, simple backup process.  Sorry but running commands via Docker or Terminal mode isn't something I wish to do.  If that's a future feature, I understand but I wouldn't risk losing my media and/or the information that goes along with each one at this point.  AI is fine and dandy but if I had to start from scratch again, I'd not be a happy camper.  Yeah, an appliance approach would be good...set a time so to speak and it just works.  Thank you.

Is such a feature available now to sponsors or on the road map?  Thank you.

**Comment by lastzero** [comment]
So you don't want to type any command? Do you have an existing backup tool? How does it work?
